### DISCUSSION TITLE
deacivation of continuously updates of docker images at 22.04

### DISCUSSION THREAD

**Discussion by arnux72** [telekom-security/tpotce#1122] [Q&A]
Hi,

how can I deactivate the new function of continuously updates of docker images at tpot version 22.04 on the easiest way? 
I need to update it manually because of using a modified logstash config file which often needs to adjust as well (e.g. changing of version numbers at pathes to database jruby after update)

Thanks a lot for our help and best regards

**Comment by t3chn0m4g3** [comment]
You can adjust `/etc/crontab` for that matter, however disabling automatic image download is not recommended.
